inference-engine/src/cldnn_engine/cldnn_graph.h

```diff
@@ -51,8 +51,10 @@ class CLDNNGraph {
     InferenceEngine::SizeVector GetOutputSize(std::string outName) const;
     std::string MapOutputName(std::string outName) const;
     std::string getName() const { return m_networkName; }
+    std::mutex* get_mutex() { return &m_infer_mutex; }
```

Comment:
I suppose you can return reference here.

Reply:
Fixed